docs/content/technical-specifications.md

```diff
@@ -23,6 +23,7 @@ We explicitly test the NGINX Ingress Controller (NIC) on a range of Kubernetes p
 {{% table %}}
 | NIC Version | Supported Kubernetes Version | NIC Helm Chart Version | NIC Operator Version | NGINX / NGINX Plus version |
 | --- | --- | --- | --- | --- |
+| 3.0.2 | 1.26 - 1.21 | 0.16.2 | 1.3.0 | 1.23.3 / R28 |
 | 3.0.1 | 1.26 - 1.21 | 0.16.1 | 1.3.0 | 1.23.3 / R28 |
```

Comment:
```suggestion
```
We only need the latest major version in this table - the script updating the previous 3.0.1 version is by design.